### PR TITLE
Refactoring gaussian_filter and gabor_filter (Ref: #1647)

### PR DESCRIPTION
**Pull request by sumitbinnani**
Refactoring ```gaussian_filter``` to ```gaussian```
Refactoring ```gabor_filter``` to ```gabor```
Reference #1647

### CODE REVIEW

File: skimage/filters/tests/test_gaussian.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def test_multichannel():
...
-        gaussian_rgb_a = gaussian_filter(a, sigma=1, mode='reflect')
+        gaussian_rgb_a = gaussian(a, sigma=1, mode='reflect')
     # Check that the mean value is conserved in each channel
     # (color channels are not mixed together)
     assert np.allclose([a[..., i].mean() for i in range(3)],
                         [gaussian_rgb_a[..., i].mean() for i in range(3)])
     # Iterable sigma
-    gaussian_rgb_a = gaussian_filter(a, sigma=[1, 2], mode='reflect',
+    gaussian_rgb_a = gaussian(a, sigma=[1, 2], mode='reflect',
                                      multichannel=True)
```
Comment (ahojnnes) [inline skimage/filters/tests/test_gaussian.py]: Make sure to adapt the indentation everywhere, please.

Comment (sumitbinnani) [inline skimage/filters/tests/test_gaussian.py]: @ahojnnes Sorry, I am unable to your statement :-(

Can you please elaborate a bit?

**Comment by sumitbinnani** [comment]
@emmanuelle : The built is failing for Python 3.2. The issue existed even when no changes were incorporated. Kindly guide.

**Comment by sumitbinnani** [comment]
@ahojnnes @stefanv 
Should I close this PR and open a new one with properly grouped commit?

**Comment by blink1073** [comment]
@sumitbinnani, you can regroup these commits by rebasing against master:

```
git fetch upstream master
git rebase -i upstream/master
```

Then, you can change `pick` to `s` or `squash` to combine commits.

**Comment by blink1073** [comment]
When you've finished, you `git push --force origin RenamingFilter`

**Comment by sumitbinnani** [comment]
@blink1073, Thanks for the help :-)

**Comment by blink1073** [comment]
No problem @sumitbinnani.  The 3.2 problem is a known issue, and we are trying to figure out what to do about it in #1740.

**Comment by sumitbinnani** [comment]
Added alias and deprecation warning for ```skimage.filters.gabor_filter``` and ```skimage.filters.gaussian_filter```. Also, updated ```TODO.txt```. 

Kindly let me know if any changes are required.

Thanks, Sumit

File: skimage/filters/tests/test_gaussian.py (hunk 1 above)
Comment (emmanuelle) [inline skimage/filters/tests/test_gaussian.py]: It's a PEP8 style point about indentation, see https://www.python.org/dev/peps/pep-0008/#indentation
Here the second line or arguments should be aligned with the first one, so ``multichannel=True`` should be just under `a`.

**Comment by sumitbinnani** [comment]
Updated edited files to comply with PEP8 Standards.

**Comment by sumitbinnani** [comment]
@blink1073 @stefanv 
The code coverage drop after making changes to comply with PEP8 Standards (may be an extra line got added due to it). Build 3.2 passed but 3.5 failed. Kindly guide.

**Comment by blink1073** [comment]
The Travis error looks to be unrelated to your PR.  I restarted it.  If it fails, I'll raise it as a separate issue.

**Comment by blink1073** [comment]
Travis is happy, looks good to me, but I'll let @stefanv press the green button.

**Comment by sumitbinnani** [comment]
Hi @blink1073 @stefanv 

Does the PR need more changes? Kindly let me know if additional changes are required :-)

**Comment by blink1073** [comment]
@stefanv, any problem with merging this?

**Comment by stefanv** [comment]
Please, go ahead.

**Comment by blink1073** [comment]
Done, thanks @sumitbinnani!

**Comment by sumitbinnani** [comment]
Happy to contribute :-)

**Pull request merged**